.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      # Prefix all commit messages with "deps: ", which should be
+      # accepted as a conventional commit and trigger release-please
+      prefix: "deps"
```

Comment:
I don't think this is necessary. Check out our [Dependabot configuration for datavzrd](https://github.com/datavzrd/datavzrd/blob/main/.github/dependabot.yml) (another software of ours). This is not specified and the PRs are titles with _build(deps):_. I don't think _deps:_ is part of the angular convention that conventional commits and release-please rely on.

```suggestion
```

Reply:
`deps:` is the convention used by both `rust-bio` and `rust-bio-types`:
https://github.com/rust-bio/rust-bio/blob/f726f3511f7286b687687b811441599b98f1dc82/.github/dependabot.yml#L7-L10
https://github.com/rust-bio/rust-bio-types/blob/e36648bd3a0f8c5cd711d060bd647f369a2c954e/.github/dependabot.yml#L7-L10

Can you please confirm that `rust-bio/rust-htslib` wishes to use a different convention `build(deps):` than the convention `deps:` used by both `rust-bio` and `rust-bio-types`?

Reply:
We can keep it like this.